services/wfm/src/test/java/org/openkilda/wfm/topology/flow/FlowTopologyTest.java

```diff
@@ -933,7 +933,7 @@ private Flow createFlow(final String flowId) throws IOException {
         System.out.println("NORTHBOUND: Create flow");
         Flow flowPayload =
                 new Flow(flowId, 10000, false, "", new SwitchId("ff:00"), 1, 2,
-                        new SwitchId("ff:00"), 1, 2);
+                        new SwitchId("ff:01"), 1, 2);
```

Comment:
?

Reply:
that's because tests try to create flow between two same endpoints (switch dpid, port, vlan) and I've just changed destination dpid.